drivers/hwinfo/hwinfo_stm32.c

```diff
@@ -42,9 +42,9 @@ ssize_t z_impl_hwinfo_get_device_id(uint8_t *buffer, size_t length)
 	sys_cache_instr_disable();
 #endif /* CONFIG_SOC_SERIES_STM32H5X */
 
```

Comment:
What about using:
```
 /* zephyr-keep-sorted-start */ 
 /* zephyr-keep-sorted-stop */ 
```

Reply:
@emillindq Ping on this proposal

Reply:
I don't know if it's possible to use it in all three places, but I pushed it now. Will the build fail if it's not sorted or the CI?